### PR TITLE
Derive `Debug` and `PartialEq` for `amethyst_input::Axis`.

### PR DESCRIPTION
**Pull request by azriel91**
Issue #903

---
This change is [Reviewable](https://reviewable.io/reviews/amethyst/amethyst/904)

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 * State::handle_event can now handle multiple types of events. ([#887])
 * Added Named Component. ([#879])([#896])
 * Support for progressive jpeg loading. ([#877])
+* Derive `Debug` and `PartialEq` for `amethyst_input::Axis`. ([#903], [#904])
```
Comment (jojolepro) [inline docs/CHANGELOG.md]: Goes in the changed category

Comment (azriel91) [reply docs/CHANGELOG.md]: changed2changed

**Comment by Moxinilian** [comment]
I think bors can handle the state of Ci but I'm not sure. We'll check.

**Pull request merged**